pkg/apis/workflow/v1alpha1/data_types.go

```diff
@@ -0,0 +1,59 @@
+package v1alpha1
```

Comment:
own file - nice

Reply:
Planning on moving the rest of the templates out in a future tech-debt PR